### ISSUE TITLE
Top 404s as of 2020/09/14

### ISSUE THREAD

**Issue by chalin** [grpc/grpc.io#392]
Here are the current top 404s (20200707-20200914):

- [x] `/2018/01/22/grpc-go-engineering-practices.html` - #394
- [x] `/blog/a_short_introduction_to_channelz` - #394
- [x] `/blog/optimizing-grpc-part-3` - #391
- [x] `/blog/grpc_on_http2` - #393
- [x] `/blog/flatbuffers` - #393
- [x] `/posts/principles` and `/posts/installation` - #393
- [x] `/docs/tutorials/auth/oauth2-objective-c.html` - #394
- `/grpc.github.io/docs/quickstart/python.html` - https://github.com/marcacohen/notes/pull/7

**Comment by chalin** [comment]
We're still waiting on https://github.com/marcacohen/notes/pull/7, but Marc communicated out-of-band that it's low priority for him -- that's fine. For the record, I haven't seen any more 404s fro `/grpc.github.io/docs/...` paths, so I'll close this issue for now.
